src/transformers/modeling_utils.py

```diff
@@ -2006,7 +2006,17 @@ def save_pretrained(
                         peft_state_dict[f"base_model.model.{key}"] = value
                     state_dict = peft_state_dict
 
-                current_peft_config = self.peft_config[self.active_adapter()]
+                active_adapter = self.active_adapter()
+
+                if isinstance(active_adapter, list):
+                    if len(active_adapter) > 1:
+                        logger.warning(
+                            "Multiple active adapters detected, will only consider the first active adapter. In order to save them all, please iteratively call `set_adapter()` on each"
+                            " adapter name and save them one by one manually. "
+                        )
+                    active_adapter = active_adapter[0]
+
+                current_peft_config = self.peft_config[active_adapter]
```

Comment:
Should multi-adapter serialization be supported out of the box?

Reply:
We could also expose a new method in `PeftAdapterMixin` `save_adapters` to support that by iteratively calling `save_pretrained` on all adapters

Reply:
Ok understood, let's go with what you have right now then